packages/vite-plugin-kit-routes/src/test/ROUTES_format-route-slash.ts

```diff
@@ -0,0 +1,169 @@
+  "/main": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/main`
+  },
+  "/match/[id=int]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/match/${params.id}`
+  },
+  "/site": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site`
+  },
+  "/site/[id]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site/${params.id}`
+  },
+  "/site_contract/[siteId]-[contractId]": (params: { siteId: (string | number), contractId: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site_contract/${params.siteId}-${params.contractId}`
+  },
+  "/a/[...rest]/z": (params: { rest: (string | number)[] }) => {
+    return `/a/${params.rest?.join('/')}/z`
+  },
+  "/lay/normal": `/lay/normal`,
+  "/lay/root-layout": `/lay/root-layout`,
+  "/lay/skip": `/lay/skip`
+}
+
+/**
+ * SERVERS
+ */
+const SERVERS = {
+  "GET /contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract`
+  },
+  "POST /contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract`
+  },
+  "GET /site": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site`
+  },
+  "GET /api/graphql": `/api/graphql`,
+  "POST /api/graphql": `/api/graphql`
+}
+
+/**
+ * ACTIONS
+ */
+const ACTIONS = {
+  "default /contract/[id]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/contract/${params.id}`
+  },
+  "create /site": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site?/create`
+  },
+  "update /site/[id]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site/${params.id}?/update`
+  },
+  "delete /site/[id]": (params: { id: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site/${params.id}?/delete`
+  },
+  "noSatisfies /site_contract": (params?: { lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site_contract?/noSatisfies`
+  },
+  "send /site_contract/[siteId]-[contractId]": (params: { siteId: (string | number), contractId: (string | number), lang?: ('fr' | 'en' | 'hu' | 'at' | string) }) => {
+    return `${params?.lang ? `/${params?.lang}`: ''}/site_contract/${params.siteId}-${params.contractId}?/send`
+  }
+}
```

Comment:
`action /the/path`, and a default action is named also

Reply:
Maybe we could skip the `default ` prefix for convenience?

Reply:
Unfortunately, with the function `route()` we can have collision quite fast. (Here with a page for example.

So, maybe I'll have to keep it. I don't really like it. 
Or maybe having the default only if collision?

Reply:
You mean the `route()` function, in its current implementation, be the single entry point for everything (pages, actions, servers, etc.) right?

Reply:
yes 👍

Reply:
(OMG, experience of comments on a PR is a disaster, I get things in wrong order... And I'm not even sure I saw everything)

Reply:
I think that collision can happen only* with the default (I could almost rename it action instead of default? )

The highest request I had was to merge everything into one object ROUTES. Then we came into the fonction 'route()' where people wanted everything.


So... I don't know... I think it's already a lot of options 😅

(and I probably want to provide one option when there is only 1 required param to give it without object!) 😳

You are somewhere to chat live? 😊